.github/workflows/doc.yml

```diff
@@ -0,0 +1,35 @@
+name: Docs
+
+on:
+  #push:
+  #  branches:
+  #  - master
+  #  - 3.8
+  #  - 3.7
+  #  paths:
+  #  - 'Doc/**'
+  pull_request:
+    branches:
+    - master
+    - 3.8
+    - 3.7
+    paths:
+    - 'Doc/**'
+    - 'Misc/**'
+
+jobs:
+  build_doc:
+    name: 'Docs'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.7'
+        architecture: 'x64'
+    - name: 'Install build dependencies'
+      run: python -m pip install sphinx==2.2.0 blurb python-docs-theme
+    - name: 'Build documentation'
+      run: |
+        cd Doc
+        make check suspicious html PYTHON=python
```

Comment:
How about upload the build html?

```
    - name: 'Build documentation'
      run: |
        cd Doc
        make check suspicious html PYTHON=python
        cd build
        tar cvzf html.tar.gz html
    - name: Upload
        uses: actions/upload-artifact@v1
        with:
          name: python-doc-html
          path: Doc/build/html.tar.gz
```

Reply:
Are the uploaded artifacts hosted somewhere to preview docs rendering? There is netlify.toml to push to netlify but I guess the integration is still not complete. https://bugs.python.org/issue37860

Reply:
> Are the uploaded artifacts hosted somewhere to preview docs rendering?

No, it is not hosted.  We can just download the tarball of the HTML.
Until we have netlify preview, the downloadable tarball is better than nothing.